packages/cloud_functions/cloud_functions/ios/Classes/FLTFirebaseFunctionsPlugin.m

```diff
@@ -93,7 +94,17 @@ - (void)httpsFunctionCall:(id)arguments withMethodCallResult:(FLTFirebaseMethodC
     [functions useEmulatorWithHost:[url host] port:[[url port] intValue]];
   }
 
-  FIRHTTPSCallable *function = [functions HTTPSCallableWithName:functionName];
+  FIRHTTPSCallable *function;
+
+  if (![functionName isEqual:[NSNull null]]) {
+    function = [functions HTTPSCallableWithName:functionName];
+  } else if (![functionUri isEqual:[NSNull null]]) {
+    function = [functions HTTPSCallableWithURL:[NSURL URLWithString:functionUri]];
+  } else {
+    result.error(@"WRONG_ARGUMENT", @"Either functionName or functionUri should be provided", nil,
```

Comment:
Sorry, I just noticed this error code: `@"WRONG_ARGUMENT"`. Perhaps we should have it matching on android as well? Also - probably make it lowercase I think to match other error code cases.

Reply:
I've updated with `IllegalArgumentException` to match Android :)

Reply:
Can this code actually happen though, shouldn't Dart side block it?

Reply:
This code shouldn't happened, it was done to handle a case if we do a mistake while programming (this could have been a else instead of a else if)